### PR TITLE
DOC 3.0 requirements: add evaluation, milestones

### PR DESCRIPTION
**Pull request by wagner-certat**

### CODE REVIEW

File: docs/architecture-3.0.md [hunk 1]
```diff
@@ -13,11 +13,17 @@ The following architecture document addresses
   principles which got lost over the last years due to feature creep
   * making IntelMQ 3.x interoperable with CERT.pl's n6 system
 
+## Implementation evaluation
+Each topic listed in this document must go through an evaulation before the final implementation and integration into IntelMQ:
+- Evaluation of use-cases and demand
+- Evaluation of technical feasibility and implementation possibilities
+- Proof-of-concept and beta version, proofing positive impact in CERT's near-production environment
```
Comment (aaronkaplan) [inline docs/architecture-3.0.md:20]: Ich think that is too complex for now. I'd rather send it to the lists.

File: docs/architecture-3.0.md [hunk 2]
```diff
@@ -13,11 +13,17 @@ The following architecture document addresses
...
+
 # Non-Scope
 
 This document will not list the individual, detailed steps which need to be taken to achieve these goals. 
 This is a high-level overview document. The detailed, individual steps will be specified as individual 
-issues on github and tagged ``intelmq3.0``.
+issues on github and assigned to the milestone `3.0`: https://github.com/certtools/intelmq/milestone/16
```
Comment (aaronkaplan) [inline docs/architecture-3.0.md:26]: +1

File: docs/architecture-3.0.md [hunk 3]
```diff
@@ -444,9 +450,7 @@ Each IntelMQ program (or -daemon) shall:
 ## Collectors
 
 Here we are already quite strong: we have lots of collectors (data importers). STIX 2.x is still missing.
-Should we get bored, here is a wish list of inputs: https://github.com/certtools/intelmq/blob/develop/docs/Feeds-whishlist.md.
-
-XXX @sebix: please also add the issues & milestone more bots feeds XXX
+Should we get bored, here is a wish list of inputs: https://github.com/certtools/intelmq/blob/develop/docs/Feeds-whishlist.md and a wishlist for bots: https://github.com/certtools/intelmq/milestone/7
```
Comment (aaronkaplan) [inline docs/architecture-3.0.md:453]: +1

**Review by aaronkaplan — COMMENTED** [review]
tbd

File: docs/architecture-3.0.md (hunk 1 above)
Comment (wagner-certat) [reply docs/architecture-3.0.md]: true, I removed it

File: docs/architecture-3.0.md [hunk 4]
```diff
@@ -13,11 +13,16 @@ The following architecture document addresses
   principles which got lost over the last years due to feature creep
   * making IntelMQ 3.x interoperable with CERT.pl's n6 system
 
+## Implementation evaluation
+The mayor changes listed in this document must go through an evaluation before the final implementation and integration into IntelMQ:
```
Comment (aaronkaplan) [inline docs/architecture-3.0.md:17]: still have a slight issue with this, could we leave it out of this document for now and let the arch. board (in future) decide how the process should be? I think that's better.

**Review by aaronkaplan — COMMENTED** [review]
ass said before, line 16 should be decided by the board

**Review by aaronkaplan — APPROVED** [review]

**Pull request merged**